src/NuGet.Core/NuGet.Build.Tasks.Console/Program.cs

```diff
@@ -188,13 +188,15 @@ internal static bool TryParseArguments(string[] args, Func<Stream> getStream, Te
                     else
                     {
 #endif
+#pragma warning disable format
```

Comment:
Due to preprocessing directives the following code was incorrectly flagged by IDE0055 analyzer (Fix formatting). Adding suppression in GlobalSuppressions.cs file didn't work hence added it directly to the code.